src/wamp-server-over-fs.ts

```diff
@@ -1,12 +1,16 @@
 import * as Promise from 'bluebird';
 import {Connection, Session} from 'autobahn';
 import {FileSystem, fileSystemEventNames, fileSystemMethods} from './api';
-
 const Router = require('wamp-server');
 
+
+export class WampConnection extends Connection {
+    isConnected: boolean
```

Comment:
that's what `custom_typings` folder is for - to contain the typing extensions of 3rd party APIs. This shouldn't be in our code.